monday/query_joins.py

```diff
@@ -59,6 +59,37 @@ def mutate_subitem_query(parent_item_id, subitem_name, column_values,
             str(create_labels_if_missing).lower())
 
 
+def mutate_multiple_items_query(items_data):
```

Comment:
Would prefer `item_data`, and `data` as it is grammatically correct and clearer than `items_data` and `item_data`